From 79fc4ec80b68eac193095ee40d5da584eb270f1c Mon Sep 17 00:00:00 2001
From: Stefan Sauer <ensonic@users.sf.net>
Date: Tue, 18 Feb 2014 21:54:45 +0000
Subject: docs: use docbook markup for xi:include

It turns out that the change in gtk-doc-1.20 which wraps the |[]| content in
CDATA break xi:inlcude examples. As in a whole jhbuild checkout these where
the only 4, we're fixing them instead.
---
diff --git a/gst/audiofx/audiofirfilter.c b/gst/audiofx/audiofirfilter.c
index e0887ad..0ab32f3 100644
--- a/gst/audiofx/audiofirfilter.c
+++ b/gst/audiofx/audiofirfilter.c
@@ -39,9 +39,9 @@
  *
  * <refsect2>
  * <title>Example application</title>
- * |[
+ * <informalexample><programlisting language="C">
  * <xi:include xmlns:xi="http://www.w3.org/2003/XInclude" parse="text" href="../../../../tests/examples/audiofx/firfilter-example.c" />
- * ]|
+ * </programlisting></informalexample>
  * </refsect2>
  */
 
diff --git a/gst/audiofx/audioiirfilter.c b/gst/audiofx/audioiirfilter.c
index c107594..eb4f20c 100644
--- a/gst/audiofx/audioiirfilter.c
+++ b/gst/audiofx/audioiirfilter.c
@@ -35,9 +35,9 @@
  *
  * <refsect2>
  * <title>Example application</title>
- * |[
+ * <informalexample><programlisting language="C">
  * <xi:include xmlns:xi="http://www.w3.org/2003/XInclude" parse="text" href="../../../../tests/examples/audiofx/iirfilter-example.c" />
- * ]|
+ * </programlisting></informalexample>
  * </refsect2>
  */
 
diff --git a/gst/level/gstlevel.c b/gst/level/gstlevel.c
index 59943d4..5cd83c8 100644
--- a/gst/level/gstlevel.c
+++ b/gst/level/gstlevel.c
@@ -95,9 +95,9 @@
  *
  * <refsect2>
  * <title>Example application</title>
- * |[
+ * <informalexample><programlisting language="C">
  * <xi:include xmlns:xi="http://www.w3.org/2003/XInclude" parse="text" href="../../../../tests/examples/level/level-example.c" />
- * ]|
+ * </programlisting></informalexample>
  * </refsect2>
  */
 
diff --git a/gst/spectrum/gstspectrum.c b/gst/spectrum/gstspectrum.c
index 75a9152..5e192c1 100644
--- a/gst/spectrum/gstspectrum.c
+++ b/gst/spectrum/gstspectrum.c
@@ -90,9 +90,9 @@
  *
  * <refsect2>
  * <title>Example application</title>
- * |[
+ * <informalexample><programlisting language="C">
  * <xi:include xmlns:xi="http://www.w3.org/2003/XInclude" parse="text" href="../../../../tests/examples/spectrum/spectrum-example.c" />
- * ]|
+ * </programlisting></informalexample>
  * </refsect2>
  *
  * Last reviewed on 2011-03-10 (0.10.29)
--
cgit v0.9.0.2-2-gbebe
